kubelet/datadog_checks/kubelet/kubelet.py

```diff
@@ -487,6 +496,32 @@ def _submit_container_state_metric(self, metric_name, state_name, c_state, state
             gauge_name = '{}.containers.{}.{}'.format(self.NAMESPACE, metric_name, state_name)
             self.gauge(gauge_name, 1, tags + reason_tags)
 
+    def _report_ephemeral_storage_usage(self, pod_list, instance_tags):
+        stats = self._retrieve_stats()
+
+        ephemeral_storage_usage = {}
+        for pod in stats['pods']:
+            pod_uid = pod.get('podRef', {}).get('uid')
+            pod_ephemeral_usage = pod.get('ephemeral-storage', {}).get('usedBytes')
+            if pod_uid is not None and pod_ephemeral_usage is not None:
+                ephemeral_storage_usage[pod_uid] = pod_ephemeral_usage
+
+        for pod in pod_list['items']:
+            pod_uid = pod.get('metadata', {}).get('uid')
+            if pod_uid is None:
+                continue
+
+            pod_usage = ephemeral_storage_usage.get(pod_uid)
+            if pod_usage is None:
+                continue
+
+            tags = tagger.tag('kubernetes_pod_uid://{}'.format(pod_uid), tagger.ORCHESTRATOR)
```

Comment:
What is the proper level of cardinality that I should use here ?
`tagger.LOW`, `tagger.ORCHESTRATOR` or `tagger.HIGH` ?

Reply:
According to @hkaj, it would be useful to go with `ORCHESTRATOR` in order to be able to drill down to the POD level.